nestkernel/source.h

```diff
@@ -40,12 +40,10 @@ namespace nest
 class Source
 {
 private:
-  uint64_t gid_ : 62;  //!< gid of source
-  bool processed_ : 1; //!< whether this target has already been moved
-                       //!to the MPI buffer
+  uint64_t gid_ : NUM_BITS_GID;  //!< gid of source
+  bool processed_ : 1;           //!< whether this target has already been moved
+                                 //!to the MPI buffer
```

Comment:
I think this line should also start with `//!<` for doxygen to parse it correctly.